test-ts/manager.test.ts

```diff
@@ -469,49 +769,43 @@ describe("Manager", () => {
     beforeEach(async () => {
```

Comment:
This test scope should be part of default-strategy.test.ts

Reply:
this is being addressed in https://github.com/celo-org/staked-celo/pull/109